package.json

```diff
@@ -101,6 +101,7 @@
     "copy-webpack-plugin": "^4.5.2",
     "coveralls": "^3.0.3",
     "cson-parser": "^1.3.3",
+    "debug": "^4.1.1",
```

Comment:
Why are you adding these to the root package.json?

Reply:
removed.

---
In reply to: [285190460](https://github.com/microsoft/azuredatastudio/pull/5499#discussion_r285190460) [](ancestors = 285190460)